### ISSUE TITLE
Can't Scroll

### ISSUE THREAD

**Issue by hnguyen94** [Floobits/atom-term3#19]
Hey I can't scroll in term3

**Comment by ggreer** [comment]
Can you scroll in term2? Basically, I want to know if this is a regression or a feature request. :)

**Comment by hnguyen94** [comment]
Yes, term2 has the same issue. When I enter the rails console then it works, but when I quit the rails console, then I can't scroll again. :)

**Comment by liudangyi** [comment]
Maybe we can use jeremyramin/term.js instead of Floobits/atom-term.js, which has fixed the zsh scroll bug.

**Comment by dustinblackman** [comment]
:+1:

**Comment by hnguyen94** [comment]
👍

**Comment by bborysenko** [comment]
:+1:

**Comment by stevenla** [comment]
seems like this commit fixed the bug https://github.com/jeremyramin/term.js/commit/5d74aa1dc5a57303d48396b6ad3a6a6bdac5b842

**Comment by ggreer** [comment]
I've cherry-picked that commit (well, the part that matters) and pushed it to https://github.com/Floobits/atom-term.js. I've also tagged a new release.
